### PR TITLE
adding fitPolicy Prop

### PR DESCRIPTION
**Pull request by kmathew-pdftron**
Adding fitPolicy prop

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -751,6 +751,17 @@ Defines the fit mode (default zoom level) of the viewer.
 />
 ```
 
+#### fitPolicy
+
+Defines the fit mode (default zoom level) of the viewer.
+0:fit both(default), 1:fit width 2:fit height
```
Comment (sgong-pdftron) [inline API.md:757]: use a table to explain the values so it is clearly presented to customers

File: android/src/main/java/com/pdftron/reactnative/views/DocumentView.java [hunk 2]
```diff
@@ -412,27 +412,43 @@ public void setReadOnly(boolean readOnly) {
...
+                getPdfViewCtrl().setPageViewMode(mode);
+            } else if (mPDFViewCtrlConfig != null) {
+                mPDFViewCtrlConfig.setPageViewMode(mode);
             }
         }
     }
 
+    public void setFitPolicy(int fitPolicy){
+        PDFViewCtrl.PageViewMode mode = null;
+        switch(fitPolicy){
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/reactnative/views/DocumentView.java:436]: code formatting (select code -> Code -> Reformat code)

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (kmathew-pdftron) [reply API.md:758]: done

File: android/src/main/java/com/pdftron/reactnative/views/DocumentView.java (hunk 2 above)
Comment (kmathew-pdftron) [reply android/src/main/java/com/pdftron/reactnative/views/DocumentView.java]: done

File: API.md (hunk 1 above)
Comment (sgong-pdftron) [reply API.md:758]: well.... with the table you can remove line 758....

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (sgong-pdftron) [reply API.md:758]: and indicate default in your table

Comment (kmathew-pdftron) [reply API.md]: done

File: API.md [hunk 3]
```diff
@@ -751,6 +751,23 @@ Defines the fit mode (default zoom level) of the viewer.
...
 ```
 
+#### fitPolicy
+
+Defines the fit mode (default zoom level) of the viewer.
+Parameters:
+
+Mode | Value | Description
+--- | --- | ---
+fit both or fitPage (default) | 0 | fits the whole page
```
Comment (sgong-pdftron) [inline API.md:761]: i don't understand what is "fit both or fitPage"

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 3 above)
Comment (kmathew-pdftron) [reply API.md:761]: it's like it fits both the height and width of the page or more like fits the page

Comment (sgong-pdftron) [reply API.md:761]: it's confusing to read, just do "fit page (default)"

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**